Header header
string child_frame_id
string intersection_name
geometry_msgs/Pose pose
string reached
